linkstatus/parser.py

```diff
@@ -42,5 +42,35 @@ def parse_file(file_path):
             line_links = parse_line(line)
             if line_links:
                 skip = True if "noqa" in line else False
-                links.append(LINKS(line=line_number + 1, urls=line_links, skip=skip))
+                links.append(LINKS(line=line_number + 1, urls=line_links, skip=skip, valid=False))
     return links
+
+
+def link_validator(links_list):
+    """Validate link
+    Args:
+        links_list: List of links.
+
+    Return:
+        Named tuple of the valid and invalid links.
+    """
+    validated_list = []
+
+    regex = re.compile(
+            r'^(?:http|ftp)s?://' # http:// or https://
```

Comment:
Thanks for submitting the PR, In windows we we have file:///? How we are going to check that?